datahub-web-react/src/graphql/dataFlow.graphql

```diff
@@ -70,7 +70,7 @@ query getDataFlow($urn: String!) {
         downstream: lineage(input: { direction: DOWNSTREAM, start: 0, count: 100 }) {
             ...partialLineageResults
         }
-        childJobs: relationships(input: { types: ["IsPartOf"], direction: INCOMING, start: 0, count: 100 }) {
```

Comment:
we could then write another query like `getDataFlowChildJobs` that takes in a `start` and `count` and use that when we want to paginate

Reply:
Done